From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: Luca Boccassi <bluca@debian.org>
Date: Wed, 19 Jul 2023 10:14:54 +0100
Subject: [PATCH] Make 'Uploaders' field optional for team maintenance

Naming people in Uploaders assigns ownership and deters team maintenance,
make the field optional
---
 policy/ch-binary.rst | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/policy/ch-binary.rst b/policy/ch-binary.rst
index a03a3b8..e314168 100644
--- a/policy/ch-binary.rst
+++ b/policy/ch-binary.rst
@@ -164,8 +164,8 @@ The format of the ``Maintainer`` control field is described in
 :ref:`s-f-Maintainer`.
 
 If the maintainer of the package is a team of people with a shared email
-address, the ``Uploaders`` control field must be present and must
-contain at least one human with their personal email address. See
+address, the ``Uploaders`` control field can be present, in which case it
+must contain at least one human with their personal email address. See
 :ref:`s-f-Uploaders` for the syntax of that field.
 
 An orphaned package is one with no current maintainer. Orphaned packages
